rosidl_typesupport_freertps_cpp/CMakeLists.txt

```diff
@@ -40,6 +41,52 @@ ament_export_libraries(${PROJECT_NAME})
 if(AMENT_ENABLE_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
+
+  # set(message_files
+  #   "msg/BoundedArrayBounded.msg"
+  #   "msg/BoundedArrayStatic.msg"
+  #   "msg/BoundedArrayUnbounded.msg"
+
+  #   "msg/Empty.msg"
+
+  #   "msg/PrimitiveStaticArrays.msg"
+
+  #   "msg/PrimitivesBounded.msg"
+  #   "msg/PrimitivesStatic.msg"
+  #   "msg/PrimitivesUnbounded.msg"
+
+  #   "msg/StaticArrayBounded.msg"
+  #   "msg/StaticArrayStatic.msg"
+  #   "msg/StaticArrayUnbounded.msg"
+
+  #   "msg/UnboundedArrayBounded.msg"
+  #   "msg/UnboundedArrayStatic.msg"
+  #   "msg/UnboundedArrayUnbounded.msg"
+  # )
+
+  # include(cmake/register_cpp.cmake)
+  # set(rosidl_typesupport_freertps_cpp_DIR "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+  # rosidl_typesupport_freertps_cpp_extras(
+  #   "${CMAKE_CURRENT_SOURCE_DIR}/bin/rosidl_typesupport_freertps_cpp"
+  #   "${CMAKE_CURRENT_SOURCE_DIR}/rosidl_typesupport_freertps_cpp/__init__.py"
+  #   "${CMAKE_CURRENT_SOURCE_DIR}/resource"
+  # )
+
+  # rosidl_generate_interfaces(${PROJECT_NAME}_test ${message_files}
+  #   SKIP_INSTALL
+  # )
+
+  # if(NOT WIN32)
+  #   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+  # endif()
+
+  # ament_add_gtest(test_serialization_cpp test/test_serialization.cpp)
+  # if(TARGET test_serialization_cpp)
+  #   add_dependencies(test_serialization_cpp ${PROJECT_NAME})
+  # endif()
+  # # include the built files directly, instead of their install location
+  # include_directories("${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}")
```

Comment:
These comments should be removed. Or based on my comment above the test should be moved to here.